packages/@o3r/configuration/schematics/add-config/schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "$id": "NgAddConfigSchematicsSchema",
+  "title": "Add Otter Configuration to existing component",
+  "description": "Add Otter Configuration to existing component",
+  "properties": {
+    "projectName": {
+      "type": "string",
+      "description": "Project name",
+      "$default": {
+        "$source": "projectName"
+      }
+    },
+    "path": {
+      "type": "string",
+      "description": "Path to the component"
+    },
+    "skipLinter": {
+      "type": "boolean",
+      "description": "Skip the linter process",
+      "default": false
+    },
+    "exposeComponent": {
+      "type": "boolean",
+      "description": "Should change a component into an exposed component",
```

Comment:
I think we should add details regarding what is an `expose component` or redirect to the documentation